packages/desktop-client/src/components/reports/graphs/tableGraph/ReportTableRow.tsx

```diff
@@ -52,19 +52,19 @@ export const ReportTableRow = memo(
           }}
           valueStyle={compactStyle}
         />
-        {item.monthData && mode === 'time'
-          ? item.monthData.map(month => {
+        {item.intervalData && mode === 'time'
+          ? item.intervalData.map(inter => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏variable naming could be improved
```suggestion
          ? item.intervalData.map(interval => {
```